plugin/src/shared/utils/update-file.ts

```diff
@@ -0,0 +1,11 @@
+import { Tree } from '@nx/devkit';
+
+export const updateFile = (
```

Comment:
we have a similar [utility](https://github.com/RonasIT/nx-generators/blob/main/plugin/src/shared/utils/cli-utils.ts#L87), can we modify it to use updateFile?